pkg/generated/bindata.go

```diff
@@ -324,6 +478,153 @@ func translationsKubectlEn_usLc_messagesK8sPo() (*asset, error) {
+"POT-Creation-Date: 2013-12-12 20:03+0000\n"
+"PO-Revision-Date: 2017-01-27 21:04-0800\n"
+"Last-Translator: Brendan Burns <brendan.d.burns@gmail.com>\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"X-Generator: Poedit 1.6.10\n"
+"X-Poedit-SourceCharset: UTF-8\n"
+"Language-Team: \n"
+"Plural-Forms: nplurals=2; plural=(n > 1);\n"
+"Language: fr\n"
+
+msgctxt "Update the annotations on a resource"
+msgid "Update the annotations on a resource"
+msgstr "Mettre à jour les annotations sur une ressource"
+
+msgctxt ""
+"watch is only supported on individual resources and resource collections - "
+"%d resources were found"
+msgid ""
+"watch is only supported on individual resources and resource collections - "
+"%d resources were found"
+msgid_plural ""
+"watch is only supported on individual resources and resource collections - "
+"%d resources were found"
+msgstr[0] ""
+"Watch est uniquement pris en charge sur les ressources individuelles et les "
+"collections de ressources -% d ressource a été trouvée"
+msgstr[1] ""
+"Montre n'est pris en charge que sur les ressources individuelles et les "
+"collections de ressources -% d ressources ont été trouvées"
+
+# https://github.com/kubernetes/kubernetes/blob/masterpkg/kubectl/cmd/apply.go#L98
+msgctxt "Apply a configuration to a resource by filename or stdin"
+msgid "Apply a configuration to a resource by filename or stdin"
+msgstr ""
+"Appliquer une configuration à une ressource par nom de fichier ou stdin"
+
+# https://github.com/kubernetes/kubernetes/blob/master/pkg/kubectl/cmd/config/config.go#L39
+msgctxt "Modify kubeconfig files"
+msgid "Modify kubeconfig files"
+msgstr "Modification des fichiers kubeconfig"
+
+# https://github.com/kubernetes/kubernetes/blob/master/pkg/kubectl/cmd/config/create_authinfo.go#L103
+msgctxt "Sets a user entry in kubeconfig"
+msgid "Sets a user entry in kubeconfig"
+msgstr "Définit une entrée utilisateur dans kubeconfig"
+
+# https://github.com/kubernetes/kubernetes/blob/master/pkg/kubectl/cmd/config/create_cluster.go#L67
+msgctxt "Sets a cluster entry in kubeconfig"
+msgid "Sets a cluster entry in kubeconfig"
+msgstr "Définit une entrée de cluster dans kubeconfig"
+
+# https://github.com/kubernetes/kubernetes/blob/master/pkg/kubectl/cmd/config/create_context.go#L57
+msgctxt "Sets a context entry in kubeconfig"
+msgid "Sets a context entry in kubeconfig"
+msgstr "Définit une entrée de contexte dans kubeconfig"
+
+# https://github.com/kubernetes/kubernetes/blob/master/pkg/kubectl/cmd/config/current_context.go#L48
+msgctxt "Displays the current-context"
+msgid "Displays the current-context"
+msgstr "Affiche le contexte actuel"
+
+# https://github.com/kubernetes/kubernetes/blob/master/pkg/kubectl/cmd/config/delete_cluster.go#L38
+msgctxt "Delete the specified cluster from the kubeconfig"
+msgid "Delete the specified cluster from the kubeconfig"
+msgstr "Supprimer le cluster spécifié à partir du kubeconfig"
```

Comment:
`à partir de` suggests *going* from a point A to a point B, rather than the intended meaning of removing something from something else.

```
Supprimer le cluster spécifié du kubeconfig
```